### DISCUSSION TITLE
请问DataGrid中如何根据单元格的值来动态修改行颜色？

### DISCUSSION THREAD

**Discussion by iceagez** [irihitech/Ursa.Avalonia#94] [Q&A]
请问DataGrid中如何根据单元格的值来动态修改行颜色？
因为Avalonia中没有DataTrigger的缘故，上述功能是否能实现，能说下思路么？
非常感谢！

**Comment by rabbitism** [comment]
https://github.com/AvaloniaUI/Avalonia/discussions/9875

可以看这个

**Comment by iceagez** [comment]
非常感谢！
